User Story1: Implementing the Student Dashboard Topaz | 1. Layout & Navigation Setup

HenrikLeutscher opened the pull request:
### What does this PR do?
Task 1 Layout: Adjusted CSS / Tailwind CSS towards the target look and responsiveness
Task 1 Navigation Setup: Has been resolved with correct navigation all over the page alongside the sidebar

### Description of Task to be completed?
Task 2: Courses & Assignments Sections
The task to create components to display list of courses and assignments

### How should this be manually tested?
1. Open the HTML on LiveServer
2. Use the inspect toolbar to view the different screen sizes
3. Click the different buttons such as the dropdown menu alongside the different "Navigation" logos and names to test navigation.

HenrikLeutscher committed:
1.Layout

Adjusted CSS / Tailwind CSS towards the target look and responsiveness

HenrikLeutscher committed:
1. Navigation Setup

This commit Task 1 Navigation Setup has been resolved with correct navigation all over the page alongside the sidebar

kevindjenhansen reviewed (changes requested):
- Make sure to use the students initials.png in the img folder for when on desktop view
- Apply the correct fonts and font bold, the "my results" and "log out" should use Roberto but it looks like you've used Poppins

HenrikLeutscher committed:
Review Changes 1 - User Story1: Implementing the Student Dashboard Topaz | 1. Layout & Navigation Setup

Changed the image according to the request and example.

Adjusted instructed fonts and sizes to be pixel perfect to the Figma Preview.

kevindjenhansen reviewed (changes requested):
I've noticed that when in mobile view the icon for the my result button isn't centered, can you fix that and then I will approve it! I will send a image to what I mean: 
![TtqhaUx](https://github.com/user-attachments/assets/64cc7bdc-6889-49d3-a201-a75b7a70b18d)

kevindjenhansen reviewed (approved):
Actually I will approve this for now since I requested the same changes in your other PR, but just keep it in mind and change it later!
